src/redux/books/books.js

```diff
@@ -0,0 +1,30 @@
+// Actions
+
+const ADDBOOK = 'books/ADDBOOK';
+const REMOVEBOOK = 'books/DELETE';
+const books = [];
+
+// Reducers
+export default function reducerBook(state = books, action = {}) {
+  switch (action.type) {
+    case ADDBOOK:
+      return [
+        ...books, action.book,
+      ];
+    case REMOVEBOOK:
+      return [
+        books.map((book) => book.id !== action.id),
+      ];
```

Comment:
- [ ] In order to remove a given book, I believe the appropriate array method would be `filter` and not map in this case. This means that you are trying to filter the books to exclude any book whose id matches the one you want to remove 👍